### PR TITLE
New Workflow SDK updates

### PR DESCRIPTION
**Pull request by ianschmitz**
This updates the Workflow TypeScript SDK instructions to suit the new and improved [Workflow SDK](https://github.com/geocortex/vertigis-workflow-sdk).

**We'll wait to merge this until Workflow 5.18 is released.**

### CODE REVIEW

File: docs/workflow/sdk-web-create-element.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+## Prerequisites
+
+[Set up](sdk-web-overview.mdx#overview) the Geocortex Workflow TypeScript SDK.
+
+## Create a Form Element
+
+1. Open up a terminal shell in the SDK folder.
+1. Run the command `npm run generate` and follow the instructions to create a new form element named `MyCustomElement`. This script creates a custom form element skeleton that you can build on with your own custom logic. The script will perform the following operations:
+    1. Create a new form element `.ts` file with the provided name in the `src/activities` folder.
```
Comment (rcooney) [inline docs/workflow/sdk-web-create-element.mdx:20]: These are **.tsx** files

File: docs/workflow/sdk-web-overview.mdx [hunk 2]
```diff
@@ -8,96 +8,126 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
...
 
-## Setup
+## Getting Started
 
-1. Open the `activity-sdk` folder found in the SDK in your favorite IDE. We recommend using Visual Studio Code for the best experience.
-1. Run `npm install` in the terminal of this folder to install the project's dependencies.
+1. Run `npx @vertigis/workflow-sdk create <library-name>` where `<library-name>` is the name of the directory that will be created in the current working directory
+    - For example: `npx @vertigis/workflow-sdk create test-activity-pack`
+1. Open the `library-name` directory in your favorite IDE. We recommend using Visual Studio Code for the best experience.
+1. Run `npm run generate` to create a new activity or form element.
```
Comment (rcooney) [inline docs/workflow/sdk-web-overview.mdx:28]: Can we add something like "Run `npm run generate` and follow the onscreen prompts to create..."

**Comment by rcooney** [comment]
`docs/workflow/sdk-web-integrating-the-arcgis-api.mdx` says "The activity-sdk uses the ArcGIS API for JavaScript 3.x by default." This isn't the case anymore.

File: docs/workflow/sdk-web-overview.mdx (hunk 2 above)
Comment (ianschmitz) [reply docs/workflow/sdk-web-overview.mdx]: Good call. Done.

File: docs/workflow/sdk-web-create-element.mdx (hunk 1 above)
Comment (ianschmitz) [reply docs/workflow/sdk-web-create-element.mdx]: Indeed they are. Whoops! Fixed.

**Review by rcooney — APPROVED** [review]

**Review by rcooney — APPROVED** [review]

File: docs/workflow/sdk-web-create-element.mdx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+## Create a Form Element
+
+1. Open up a terminal shell in the SDK folder.
+1. Run the command `npm run generate` and follow the instructions to create a new form element named `MyCustomElement`. This script creates a custom form element skeleton that you can build on with your own custom logic. The script will perform the following operations:
+    1. Create a new form element `.tsx` file with the provided name in the `src/activities` folder.
+    1. Populate the form element `.tsx` file from a form element template.
+    1. Exports the element in `src/index.ts` to make it available for use.
+
+:::tip
+It's convention to use PascalCase for activity names.
```
Comment (kenlyon) [inline docs/workflow/sdk-web-create-element.mdx:25]:
```suggestion
It's convention to use PascalCase for form element names.
```

**Review by kenlyon — APPROVED** [review]

**Review by rcooney — APPROVED** [review]
Looks good.

**Pull request merged**